extras/nginx/nginx-1.16.patch

```diff
@@ -1849,6 +1846,8 @@ index 000000000..972ecd223
 +        return;
 +    }
 +
++    stream->in_closed = !quiche_h3_event_header_has_body(ev);
```

Comment:
In general I agree with API changes to report and of header separately, but why is have `has_body` suffix? It confuses me which body of header (because http header doesn't have a body...) so I suggest something like 'quiche_h3_event_header_finished()` to be clear with the intention (stop reading headers). I think it used to be `fin` but curious why the name is changed.

Reply:
The method name is a bit clunky but it basically allows for a C client to fetch the field from Rust Event::Headers structure that is documented in rustdoc.

Conceptually, we are trying to articulate whether the stream has more data on it, so applications can decide what to do (if anything).  The problem with the field name fin was that it is confusable with Event::Finished.